config/v1/feature_gates.go

```diff
@@ -193,7 +193,7 @@ var (
 		OwningProduct:       kubernetes,
 	}
 
-	FeatureGateEventedPLEG = FeatureGateName("EventedPLEG")
+	FeatureGateEventedPLEG = FeatureGateName("EventedPleg")
```

Comment:
I don't think changing this really solves the issue does it? Your intention is to enable the EventedPLEG feature at some point via turning the feature gate on in Kubelet, we've observed it has issues today with our existing configuration, how will you ensure in the future that it doesn't break customer clusters if you are enabling it?

I'm not sure why masking it now is better than digging out the root cause of the issue we are observing and working out what the incompatibility is, seems we will need to work it out as part of enabling this feature

Reply:
@JoelSpeed , This change is purely part of the debugging purpose and there is no intention of masking the issue to pass the tests.
I want to enable the evented pleg feature only in either of kubelet, cri-o at once and see if I get any clue on the issue.
So, please ignore these changes until I get back with the root cause analysis !!